op-challenger/fault/position_test.go

```diff
@@ -95,3 +96,14 @@ func TestTraceIndex(t *testing.T) {
 		require.Equal(t, test.TraceIndex, result)
 	}
 }
+
+// TestPosition_GetParentGIndex_Tabled tests the GetParentGIndex function on the treeNodesMaxDepth4 data.
+// The parent index of a node is the floor of the node's index divided by 2 (since the first index is 1).
+func TestPosition_GetParentGIndex_Tabled(t *testing.T) {
```

Comment:
no need for the `_Tabled` comment. TBH parent gindex should be added to the table & we should manually assert that specific values are correct.